examples/scripts/example32-total-rows.js

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import ReactDataGrid from 'react-data-grid';
+import exampleWrapper from '../components/exampleWrapper';
+
+const columns = [
+  {
+    key: 'id',
+    name: 'ID',
+    frozen: true
+  },
+  {
+    key: 'task',
+    name: 'Title',
+    width: 200,
+    frozen: true,
+    formatter: ({ value, row: { selectedIndexes}, isBottomPinned }) => {
+      if (isBottomPinned) {
+        return <strong>{ selectedIndexes ? `${selectedIndexes.length} rows selected` : 'Total'}</strong>;
+      }
+
+      return <div>{value}</div>
+    }
+  },
+  {
+    key: 'priority',
+    name: 'Priority',
+    width: 200
+  },
+  {
+    key: 'issueType',
+    name: 'Issue Type',
+    width: 200
+  },
+  {
+    key: 'complete',
+    name: '% Complete',
+    width: 200
+  },
+  {
+    key: 'startDate',
+    name: 'Start Date',
+    width: 200
+  },
+  {
+    key: 'completeDate',
+    name: 'Expected Complete',
+    width: 200
+  },
+  {
+    key: 'cost',
+    name: 'Resource cost (USD)',
+    width: 200,
+    formatter: ({ value }) => <div>{ value.toLocaleString('en-US') }</div>
+  },
+  {
+    key: 'hours',
+    name: '# of working hours',
+    width: 200,
+    formatter: ({ value }) => <div>{ value.toLocaleString('en-US', { maximumFractionDigits: 1 }) }</div>
+  },
+  {
+    key: 'issueCount',
+    name: '# of issues',
+    width: 200,
+    formatter: ({ value }) => <div>{ value.toLocaleString('en-US', { maximumFractionDigits: 0 }) }</div>
+  },
+];
+
+function getRandomDate(start, end) {
+  return new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime())).toLocaleDateString();
+};
+
+function Example() {
+  const [selectedIndexes, setSelectedIndexes] = useState([]);
+
+  const rows = useMemo(() => {
+    const rows = [];
+
+    for (let i = 1; i < 1000; i++) {
+      rows.push({
+        id: i,
+        task: `Task ${i}`,
+        complete: Math.min(100, Math.round(Math.random() * 110)),
+        priority: ['Critical', 'High', 'Medium', 'Low'][Math.floor((Math.random() * 3) + 1)],
```

Comment:
```suggestion
        priority: ['Critical', 'High', 'Medium', 'Low'][Math.round((Math.random() * 3) + 1)],
```
We won't see any `Critical` otherwise, same for the line below.